src/page-modules/assistant/server/journey-planner/index.ts

```diff
@@ -55,14 +55,16 @@ export function createJourneyApi(
         name: input.to.name,
       };
 
-      const when = 'arriveBy' in input ? input.arriveBy : input.departBy;
+      const when = input.departureDate
+        ? new Date(input.departureDate)
+        : new Date();
```

Comment:
Would it be clearer if this was a Date type all the way through? So String when posting to HTTP request, but inside the application it is Date (serialize / deserialize "on edges of app" as I wrote somewhere else)